C/plugins/storage/postgres/init.sql

```diff
@@ -925,6 +925,12 @@ NULL, '00:00:00', true);
 -- values ('543a59ce-a9ca-11e7-abc4-cec278b6b50a', 'device', 'CC2650POLL', 1,
 -- '0:0', true);
 
+-- Start the async mode CC2650 Sensortag at start-up
+insert into foglamp.schedules(id, schedule_name, process_name, schedule_type,
+schedule_interval, exclusive)
+values ('716a16ea-c736-490b-86d5-10204585ca8c', 'device', 'CC2650ASYN', 1,
+'0:0', true);
+
```

Comment:
I think we should leave this commented out for now, since we have a conflict with the poll version. As soon as we add the change to disable schedules then we can have both and make then disabled by default.